website/templates/project/project_base.mako

```diff
@@ -1,5 +1,25 @@
 <%inherit file="../base.mako"/>
 
+<%def name="resource()"><%
+        if node.get('is_registration', False):
+           return 'registration'
+        elif node.get('is_preprint', False):
+           return 'preprint'
+        elif parent_exists:
+            return 'component'
```

Comment:
Components show up as "projects".  My guess is that this method doesn't have access to parent_exists.  You might need to redefine here ` parent_exists = parent_node['exists']`.